api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -15,51 +15,57 @@
 @Table(name = "institution_email_address")
 public class DbInstitutionEmailAddress {
 
-  @Id
-  @GeneratedValue(strategy = GenerationType.IDENTITY)
-  @Column(name = "institution_email_address_id")
   private long institutionEmailAddressId;
-
-  @ManyToOne(fetch = FetchType.LAZY)
-  @JoinColumn(name = "institution_id")
   private DbInstitution institution;
-
-  @Column(name = "email_address", nullable = false)
   private String emailAddress;
 
   public DbInstitutionEmailAddress() {}
 
-  public DbInstitutionEmailAddress(DbInstitution institution, String emailAddress) {
-    this.institution = institution;
-    this.emailAddress = emailAddress;
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_email_address_id")
+  public long getInstitutionEmailAddressId() {
```

Comment:
If this is nullable, should we return `Optional<Long>`?

Reply:
It is not nullable.  Thanks for the catch.

Reply:
Oh I thought it was boxed. nvm.